airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -68,6 +68,11 @@
 import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 
+/**
+ * The web backend is an abstraction that allows the frontend to structure data in such a way that
+ * it is easier for a react frontend to consume. It should NOT have direct access to the database.
+ * It should operate exclusively by calling other endpoints that are exposed in the API.
```

Comment:
I think that the last sentence is missleading because we are not calling other endpoints but other handler. Moving the calls we do to the repository to another handler will still add a direct call to the DB. If we want to call the other endpoint through this handler we should use the generated clients via open API (like `io.airbyte.api.client.generated.HealthApi`). I don't think that calling the repository in this class will prevent us from having the server to be the gateway of all of our DB access since this handler is not use outside of the server module.